redbot/core/core_commands.py

```diff
@@ -4319,42 +4296,35 @@ async def command_disable_guild(self, ctx: commands.Context, *, command: str):
         **Arguments:**
             - `<command>` - The command to disable for the current server.
         """
-        command_obj: Optional[commands.Command] = ctx.bot.get_command(command)
-        if command_obj is None:
-            await ctx.send(
-                _("I couldn't find that command. Please note that it is case sensitive.")
-            )
-            return
-
-        if self.command_manager in command_obj.parents or self.command_manager == command_obj:
+        if self.command_manager in command.parents or self.command_manager == command:
             await ctx.send(
                 _("The command to disable cannot be `command` or any of its subcommands.")
             )
             return
 
-        if isinstance(command_obj, commands.commands._RuleDropper):
+        if isinstance(command, commands.commands._RuleDropper):
             await ctx.send(
                 _("This command is designated as being always available and cannot be disabled.")
             )
             return
 
-        if command_obj.requires.privilege_level > await PrivilegeLevel.from_ctx(ctx):
+        if command.requires.privilege_level > await PrivilegeLevel.from_ctx(ctx):
             await ctx.send(_("You are not allowed to disable that command."))
             return
 
         async with ctx.bot._config.guild(ctx.guild).disabled_commands() as disabled_commands:
             if command not in disabled_commands:
```

Comment:
`command` used to be a string so this needs to be updated appropriately:
```suggestion
            if command.qualified_name not in disabled_commands:
```